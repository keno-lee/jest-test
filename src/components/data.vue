<template>
  <div class="hello">
    <h1>{{ msg }}</h1>
    <input type="text" v-model="num">
  </div>
</template>

<script>
export default {
  name: 'HelloWorld',
  props: {
    msg: String
  },
  data() {
    return {
      num: 0
    }
  }
}
</script>
