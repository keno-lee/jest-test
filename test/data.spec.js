import { shallowMount } from '@vue/test-utils'
import data from '../src/components/data.vue'

describe('data', () => {
  test('是一个 data 组件', () => {
    const wrapper = shallowMount(data)
    expect(wrapper).toMatchSnapshot()
  })
})