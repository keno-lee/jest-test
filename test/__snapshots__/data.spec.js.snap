// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`data 是一个 data 组件 1`] = `
<div class="hello">
  <h1></h1> <input type="text">
</div>
`;
